### PR TITLE
Custom rspec formatter,rake tasks

### PR DESCRIPTION
**Pull request by VikaZinovieva**
-made custom rspec formatter
-add rake tasks

### CODE REVIEW

File: tests/.rspec [hunk 1]
```diff
@@ -0,0 +1,3 @@
+--require ./support/rspec_formatter.rb
+--format RspecFormatter
+--out reports/results/rspec<%= ENV["TEST_ENV_NUMBER"] %>.xml
```
Comment (olichka12) [inline tests/.rspec:3]: Add one more line

File: tests/Rakefile [hunk 2]
```diff
@@ -0,0 +1,38 @@
+require 'rspec/core/rake_task'
+require 'parallel_tests'
+require 'parallel_tests/tasks'
+require 'rake'
+require 'parallel_tests/cli'
+
+RSpec::Core::RakeTask.new(:all_spec) do |t|
```
Comment (olichka12) [inline tests/Rakefile:7]: Can be in {}

File: tests/Rakefile [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+  all_tasks << create_task.name
+  create_task.rspec_opts = "--format h > reports/#{ENV['METHOD']}/#{task_name}.html"
+end
+multitask :all_tests => all_tasks
+
+RSpec::Core::RakeTask.new(:get_location) do |t|
+  t.pattern = Dir.glob('spec/locations/api_get_locations_spec.rb')
+end
+
+task :parallel_rspec do |t, args|
```
Comment (olichka12) [inline tests/Rakefile:34]: In 11 row, you have the same rake task
Could you remove one of them?

File: tests/support/rspec_formatter.rb [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+    formatted_message = strip_message_from_whitespace(example.execution_result.exception.message)
+    "#{full_description} - #{location} \n  #{formatted_message}"
+  end
+
+  def strip_message_from_whitespace(msg)
+    msg.split("\n").map(&:strip).join("\n#{add_spaces(10)}")
+  end
+
+  def add_spaces n
+    " " * n
```
Comment (olichka12) [inline tests/support/rspec_formatter.rb:56]: Edit to single quotes

File: tests/reports/delete/api_delete_employees_spec.html [hunk 5]
```diff
@@ -0,0 +1,319 @@
+<!DOCTYPE html>
```
Comment (olichka12) [inline tests/reports/delete/api_delete_employees_spec.html:1]: tests/reports can be added to gitignore

**Review by olichka12 — CHANGES_REQUESTED** [review]

**Comment by VikaZinovieva** [comment]
Invalid project structure

**Pull request closed without merging**